main/ertp/guide/math-helpers.md

```diff
@@ -31,6 +48,5 @@ information. An amount might look like:
 { brand: myBrand, extent: [{ seat: '16F', flight: '39DFK', date: '2020-06-08'}] }
 ```
 
-While the `set` mathHelpers are slower than the `strSet`, they are
-more trustworthy because all of the necessary information is in the
-extent.
+While the `set` mathHelpers are slower than the `strSet` equivalents, they are
```

Comment:
I don't understand the purpose of this statement. Why is it helpful to say that there's a performance difference? What do we mean by saying "all the necessary information is in the extent" of one but not the other?

Reply:
We can probably back off of claims about performance, because I implemented a performance enhancement for `set` so it may not actually be much worse than `strSet` performance. 

> What do we mean by saying "all the necessary information is in the extent" of one but not the other?

By this, I meant that in `strSet` extents, often you only get the ID (the VIN for cars, to use an analogy), and other important information is stored elsewhere (the car make, model, color, etc.) In `set`, you often put all the information that is important for a third party to know in the extent. Information that is in an immutable extent is more credible than information that is retrieved from an external source that could potentially manipulate the information without anyone realized.

I think we want to keep these kinds of considerations in this document because we want to be moving towards documentation that is less factual and more helpful.

Reply:
What is it about `str` vs. `strSet` that would lead people to use them in these distinct ways? I could easily imagine other developers using them in ways that don't follow this pattern. 

How about "amounts are more trustworthy when all the necessary information is in the extent"